src/frontend/src/components/Footer.tsx

```diff
@@ -52,10 +52,11 @@ function CopyRight() {
     const Bitbucket = () => <Link to="https://bitbucket.org/" className="hover:underline">Bitbucket</Link>;
     const Gitlab = () => <Link to="https://about.gitlab.com/" className="hover:underline">Gitlab</Link>;
 
-    return <div>
-        <p className="text-center">
+    return <div className="text-center">
+        <p>
             <Gitalytics /> is a tool to analyze all your Repositories from <Github />, <Bitbucket /> and <Gitlab />
         </p>
+        <small>&copy; Copyright { new Date().getFullYear() }, <Gitalytics />.org</small>
```

Comment:
```suggestion
        <small>&copy; Copyright { new Date().getFullYear() }, <Gitalytics /></small>
```
Wir heißen ja als Organization Gitalytics.org. Wollen wir das in die Komponente mit reinnehmen?